public/js/add_tables.js

```diff
@@ -0,0 +1,22 @@
+$(document).ready(function(){
+
+  var table_count = 0;
+  var table_track = {};
```

Comment:
Identifier 'table_track' is not in camel case.<br>'table_track' is defined but never used.